api/src/org/labkey/api/security/UserManager.java

```diff
@@ -619,6 +619,13 @@ public static void updateLogin(User user)
         new SqlExecutor(CORE.getSchema()).execute(sql);
     }
 
+    /** Clear the ExpirationDate field for the given user */
+    public static void clearExpirationDate(User currentUser, User user)
```

Comment:
The param names aren't super clear. Just reading this method I can't tell which user is being updated.
```suggestion
    public static void clearExpirationDate(User executor, User expiringUser)
```

Reply:
Improved in 4c37b77